agent/agents/process/process_logger.go

```diff
@@ -116,6 +117,27 @@ func replacer(redactWords []string) *strings.Replacer {
 	return strings.NewReplacer(r...)
 }
 
+var matchLogLevelRegex = regexp.MustCompile(`level=(\w+)`)
+
+func matchLogLevel(line string) logrus.Level {
+	matches := matchLogLevelRegex.FindStringSubmatch(line)
+	if len(matches) < 2 {
+		return logrus.InfoLevel
+	}
+
+	result, err := logrus.ParseLevel(matches[1])
+	if err != nil {
+		// unreachable
+		return logrus.ErrorLevel
+	}
+
+	if result < logrus.ErrorLevel {
+		return logrus.ErrorLevel
+	}
+
+	return result
+}
+
```

Comment:
could you add tests for this method, please?

Reply:
@BupycHuk done